README.md

```diff
@@ -0,0 +1,370 @@
+
+There is a Private Root CA and a Private Issuing CA.  The Private Issuing CA was used for issuing of the private code signing certs that are found in the SHIM.
+
+Those issued certs are then stored on a physical HSM.  That HSM is installed within a FIPS environment.  All access to that environment is strictly controlled with physical and logical controls in place, with no outside access permitted.  The servers are in a locked environment and within a secure data center with proper physical access controls in place at that location for security purposes.
+
+*******************************************************************************
+### Do you use EV certificates as embedded certificates in the SHIM?
+*******************************************************************************
+No, only the CIQ secureboot CA (PKI) is embedded in our Shim
+
+*******************************************************************************
+### Do you add a vendor-specific SBAT entry to the SBAT section in each binary that supports SBAT metadata ( GRUB2, fwupd, fwupdate, systemd-boot, systemd-stub, UKI(s), shim + all child shim binaries )?
+### Please provide exact SBAT entries for all shim binaries as well as all SBAT binaries that shim will directly boot.
+### Where your code is only slightly modified from an upstream vendor's, please also preserve their SBAT entries to simplify revocation.
+If you are using a downstream implementation of GRUB2 or systemd-boot (e.g.
+from Fedora or Debian), please preserve the SBAT entry from those distributions
+and only append your own. More information on how SBAT works can be found
+[here](https://github.com/rhboot/shim/blob/main/SBAT.md).
+*******************************************************************************
+
+```
+sbat,1,SBAT Version,sbat,1,https://github.com/rhboot/shim/blob/main/SBAT.md
+grub,3,Free Software Foundation,grub,2.06,https//www.gnu.org/software/grub/
+grub.rh,2,Red Hat,grub2,2.06-92.el9,mailto:secalert@redhat.com
+grub.rocky,2,Rocky Linux,grub2,2.06-92.el9,mailto:security@rockylinux.org
+grub.ciq_rocky,1,Rocky Linux (CIQ modified),grub2,2.06-92.el9_2.92ciq_lts,mailto:secureboot@ciq.com
+
+objcopy --only-section .sbat -O binary fwupdaa64.efi /dev/stdout
+sbat,1,UEFI shim,sbat,1,https://github.com/rhboot/shim/blob/main/SBAT.md
+fwupd-efi,1,Firmware update daemon,fwupd-efi,1.4,https://github.com/fwupd/fwupd-efi
+fwupd-efi.rhel,1,Red Hat Enterprise Linux,fwupd,1.9.13,mail:secalert@redhat.com
+fwupd-efi.rocky,1,Rocky Linux,fwupd,1.9.13,mail:security@rockylinux.org
+fwupd-efi.ciq_rocky,1,Rocky Linux (CIQ modified),fwupd,1.9.13,mail:secureboot@ciq.com
+
+objcopy --only-section .sbat -O binary shimaa64.efi /dev/stdout
+sbat,1,SBAT Version,sbat,1,https://github.com/rhboot/shim/blob/main/SBAT.md 
+shim,4,UEFI shim,shim,1,https://github.com/rhboot/shim 
+shim.ciq,1,Ctrl IQ Inc,shim,15.8,mail:it_security@ciq.com
```

Comment:
I believe the SBAT was updated to

![image](https://github.com/user-attachments/assets/cf18245c-e33c-41cc-8e49-f86ce372d217)

We might need to check into this one.

Reply:
I think the shim package in the test repo needs to be updated.

The screen shot above is what is in the unsigned RPM package.